models/new_model

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+
+model_name=$1
+location_mod=$2
+
+if [[ ("$1" = "help") || ("$1" = "--help") || ($# -eq 0) ]]; then
+  echo "Usage: $0 [model_name] [location_mod]"
+  exit 0
+elif [[ -d "$model_name" ]]; then
+  echo "$model_name already exists! Please try a different model name."
+  exit 1
+elif [[ $# -eq 1 ]]; then
+  echo "Please enter a proper location! (Hint: check DART/assimilation_code/location for location modules avaliable.)"
```

Comment:
If `$# -eq 1` this could be the user has given a location, but not a model_name.
If the arguments are not what is expected, best to print the usage

Reply:
Agree, or they might have entered something wrong e.g., `./new_model -help`